chalice-icon-theme.json

```diff
@@ -233,7 +233,8 @@
     "tf": "script.inverse",
     "kt": "code.inverse",
     "kts": "code.inverse",
-    "astro": "bracket.inverse"
+    "astro": "bracket.inverse",
+    "sol": "code.normal",
```

Comment:
It should be "code.inverse".  Because if the user has a dark theme selected, the icons should be light.